### PR TITLE
Add a default size for images

### PR DESCRIPTION
**Pull request by towanda**

### CODE REVIEW

File: lib/article_json/export/amp/elements/image.rb [hunk 1]
```diff
@@ -6,6 +6,15 @@ class Image < Base
           include Shared::Caption
           include Shared::Float
 
+          # @param [ArticleJSON::Elements::Base] element
+          # @param String width
+          # @param String height
+          def initialize(element, width: '640px', height: '480px')
```
Comment (dsager) [inline lib/article_json/export/amp/elements/image.rb]: I think it should not have the `px` in the value, all the examples don't have it (as a normal `img` tag wouldn't):
https://www.ampproject.org/docs/reference/components/amp-img

Comment (nicolas-fricke) [reply lib/article_json/export/amp/elements/image.rb]: Also: where do these values come from and how can they be overwritten? 

If they shouldn't be the AMP-standard values, making them configurable through the gem config might be good..

Comment (towanda) [reply lib/article_json/export/amp/elements/image.rb]: @dsager right, changing it

Comment (towanda) [reply lib/article_json/export/amp/elements/image.rb]: @nicolas-fricke  we still don't know, the thing is that for AMP is mandatory to specify the image width and height, and we don't store it in the json structure yet. 

So the plan was to hardcore the default values we use in devex for the images, have one implementation working and then think about how to make it configurable. But I thought having an initializer could be nicer and more useful than having it just hardcored.

**Comment by dsager** [comment]
@towanda you forgot to change the tests :)

**Comment by towanda** [comment]
@dsager changed, I need a coffe :)

**Comment by dsager** [comment]
@towanda 
Now that I think about it, I don't like the initializer options. Because it's the only element that overwrites the initializer, and the factory pattern we are using in the base class doesn't know about it. I would just add height/width methods that, for now, return the hard coded values. we can then later enhance it...
What do you think?

**Comment by towanda** [comment]
@dsager I see your point, and I think it's ok for now, but at some point we'll need to implement a way to specify different sizes per image.

**Comment by dsager** [comment]
Yes, but it should be part of the image element, not just the exporter. Like I said, I would just hard code it now - we can then later add size info to the format itself or have the default configurable. Important is, that before displaying, we'll scale the images accodingly using cloudinary.

**Comment by towanda** [comment]
@dsager changed, thanks for the comments

**Comment by dsager** [comment]
+1
Thanks!

**Comment by yaneq** [comment]
+1

**Pull request merged**